services/src/test/scala/cromwell/services/auth/ecm/EcmConfigSpec.scala

```diff
@@ -13,10 +13,10 @@ class EcmConfigSpec extends AnyFlatSpec with Matchers {
                                               |ecm.base-url = "https://mock-ecm-url.org"
       """.stripMargin)
 
-    val actualEcmConfig = EcmConfig.apply(config)
+    val actualEcmConfig = EcmConfig(config)
 
-    actualEcmConfig shouldBe defined
-    actualEcmConfig.get.baseUrl shouldBe "https://mock-ecm-url.org"
+    actualEcmConfig.baseUrl shouldBe defined
+    actualEcmConfig.baseUrl.get shouldBe "https://mock-ecm-url.org"
   }
 
   it should "return None when ECM base url is absent" in {
```

Comment:
Should we also have a case for the other config values (to continue the theme... like `enabled`!)

Reply:
Yes they should be covered [here](https://github.com/broadinstitute/cromwell/pull/7392/files#diff-5af26f8177b51a07147f7e4eb5db049a8a8d3b2f32c58ff7d09d46f87cd397f8R47)